.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 
 # Common executor configuration
 executors:
-  clojure:
```

Comment:
You might be able to simplify this config using [matrix jobs](https://circleci.com/blog/circleci-matrix-jobs/).

Reply:
Thanks for the heads up, I'll play around with that later.